fastfilter/src/main/java/org/fastfilter/bloom/BlockedBloom.java

```diff
@@ -12,8 +12,8 @@
 public class BlockedBloom implements Filter {
 
     public static BlockedBloom construct(long[] keys, int bitsPerKey) {
-        long n = keys.length;
```

Comment:
Yes, using long doesn't make sense here... I used long in some places to avoid integer overflow, but here it's not needed.

Reply:
In cases where you can reason that overflow is impossible, accumulating into a `long` can cause significant degradation in throughput once the loop has been compiled by C2. I can share some benchmarks which demonstrate this. Of course, sometimes it's the only safe thing to do...

Reply:
Hm, I wasn't aware of this... I will keep this in mind!